cmd/snap/cmd_snap_op.go

```diff
@@ -202,18 +203,24 @@ func refreshAll() error {
 	if err != nil {
 		return fmt.Errorf("cannot list updates: %s", err)
 	}
+	// nothing to update/list
+	if len(updates) == 0 {
+		return nil
```

Comment:
one approach would be to do different things if the output is a tty vs not mmh, or otherwise I also wonder whether those outputs when empty should go to stderr instead of stdout